.github/workflows/remove-preview.yml

```diff
@@ -0,0 +1,41 @@
+name: Remove preview
+on:
+  pull_request:
+    types:
+      - closed
+jobs:
+  remove:
+    name: "Remove preview build"
+    runs-on: ubuntu-20.04
+    concurrency:
+      group: ovirt-site-preview-build
+      cancel-in-progress: false
+    steps:
+    - name: Checkout gh-actions branch
+      uses: actions/checkout@v2
+      with:
+        ref: gh-pages
+    - name: Comment on PR
+      run: |
+        set -euo pipefail
+        PR_ID=$(echo $GITHUB_REF | awk 'BEGIN { FS = "/" } ; { print $3 }')
```

Comment:
PR-ID is recalculated so many times and sometimes even inlined which does not help reading the code.
Maybe we could consider using one of the methods highlighted in: https://stackoverflow.com/questions/57819539/github-actions-how-to-share-a-calculated-value-between-job-steps
(as you did for targetdir)

Reply:
Done.